### PR TITLE
Anagram of String in C

### PR DESCRIPTION
**Pull request by AakritiKinra**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

### Description

This is a C program to check if two input strings are anagrams 

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

#860

### CODE REVIEW

**Review by iamrajiv — CHANGES_REQUESTED** [review]
@AakritiKinra remove unnecessary line breaks and indentations. Also, add space and time complexity

**Comment by HarshCasper** [comment]
Edit the `README` file

**Review by Kajol-Kumari — APPROVED** [review]
@AakritiKinra Functionality is working fine! Codewise it's good to me. Just update the `readme` and it can be merged!

**Comment by AakritiKinra** [comment]
I have edited the readme file

**Review by iamrajiv — APPROVED** [review]

File: anagram_program.c [hunk 1]
```diff
@@ -0,0 +1,64 @@
+#include <stdio.h>
```
Comment (atarax665) [inline anagram_program.c:3]: Add program description at the top and update README.md to get this PR merged.

**Review by atarax665 — CHANGES_REQUESTED** [review]

File: anagram_program.c (hunk 1 above)
Comment (AakritiKinra) [reply anagram_program.c:3]: I have added the program description however I am a bit confused about how to edit the README.md file. Could you please guide me.

**Review by Abhijit2505 — APPROVED** [review]
Looking Good To Me

**Pull request merged**